tests/Test/Prometheus/APCu/APCuTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Test\Prometheus\APCu;
+
+use Prometheus\Storage\APCu;
+use RuntimeException;
+
+final class APCuTest extends \PHPUnit_Framework_TestCase
+{
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (extension_loaded('apcu')) {
+            $this->markTestSkipped('apcu extension already available');
+            return;
+        }
+
+        if (class_exists(APCUIterator::class)) {
```

Comment:
This too can be simplified by adding `@requires function APCUIterator::__construct` on the class

Reply:
scenario here is the opposite